node/src/service.rs

```diff
@@ -592,21 +592,22 @@ where
                 ))),
                 create_inherent_data_providers: move |block: Hash, ()| {
                     let current_para_block = client_for_cidp
-                    .number(block)
-                    .expect("Header lookup should succeed")
-                    .expect("Header passed in as parent should be present in backend.");
+                    	.number(block)
+                    	.expect("Header lookup should succeed")
+                    	.expect("Header passed in as parent should be present in backend.");
 
                     let hash = client
                         .hash(current_para_block.saturating_sub(1))
                         .expect("Hash of the desired block must be present")
                         .expect("Hash of the desired block should exist");
 
-                    let para_header = client
-                        .expect_header(hash)
-                        .expect("Expected parachain header should exist")
-                        .encode();
+					let para_header = client
+						.expect_header(hash)
+						.expect("Expected parachain header should exist")
+						.encode();
 
-                    let para_head_data = HeadData(para_header).encode();
+                    let raw_para_head_data = HeadData(para_header);
+					let para_head_data = raw_para_head_data.encode();
```

Comment:
Few too many tabs

Reply:
That's weird, it looks totally normal locally, not sure what happened here

Reply:
That's weird, it looks totally normal locally, not sure what happened here